### PR TITLE
enlarge text for >=1440p resolutions

### PR DESCRIPTION
**Pull request by brysondev**
## Description

Upscale font sizes for high-resolution monitors
## Toolchain

- Windows MSVC VS2022

Preview:

4K Options page
4K Main Menu
4K In-Game Chat

### CODE REVIEW

File: mp/game/neo/resource/ClientScheme.res [hunk 1]
```diff
@@ -1155,7 +1164,27 @@ Scheme
...
+				"antialias"	"1"
+				"additive"	"1"
+			}
+			"6"
+			{
+				"name"		"Neuropol2"
+				"tall"		"26"
+				"weight"	"600"
+				"range"		"0x0000 0x017F" //	Basic Latin, Latin-1 Supplement, Latin Extended-A
+				"yres"	"1536 1599"
```
Comment (AdamTadeusz) [inline mp/game/neo/resource/ClientScheme.res:1177]: shouldn't this start at 1441?

File: mp/game/neo/resource/ClientScheme.res [hunk 2]
```diff
@@ -1202,7 +1231,25 @@ Scheme
...
+				"yres"	"1200 1440"
+				"antialias"	"1"
+			}
+			"6"
+			{
+				"name"		"Neuropol2"
+				"tall"		"26"
+				"weight"	"600"
+				"range"		"0x0000 0x017F" //	Basic Latin, Latin-1 Supplement, Latin Extended-A
+				"yres"	"1536 1599"
```
Comment (AdamTadeusz) [inline mp/game/neo/resource/ClientScheme.res:1243]: ditto

File: mp/game/neo/resource/ClientScheme.res [hunk 3]
```diff
@@ -1253,7 +1300,27 @@ Scheme
...
+				"antialias"	"1"
+				"additive"	"1"
+			}
+			"6"
+			{
+				"name"		"Neuropol2"
+				"tall"		"30"
+				"weight"	"600"
+				"range"		"0x0000 0x017F" //	Basic Latin, Latin-1 Supplement, Latin Extended-A
+				"yres"	"1536 1599"
```
Comment (AdamTadeusz) [inline mp/game/neo/resource/ClientScheme.res:1313]: ditto

File: mp/game/neo/resource/ClientScheme.res [hunk 4]
```diff
@@ -1300,7 +1367,25 @@ Scheme
...
+				"yres"	"1200 1440"
+				"antialias"	"1"
+			}
+			"6"
+			{
+				"name"		"Neuropol2"
+				"tall"		"30"
+				"weight"	"600"
+				"range"		"0x0000 0x017F" //	Basic Latin, Latin-1 Supplement, Latin Extended-A
+				"yres"	"1536 1599"
```
Comment (AdamTadeusz) [inline mp/game/neo/resource/ClientScheme.res:1379]: you get the idea

File: mp/game/neo/resource/ClientScheme.res [hunk 5]
```diff
@@ -1353,7 +1438,27 @@ Scheme
...
+				"antialias"	"1"
+				//"additive"	"1"
+			}
+			"6"
+			{
+				"name"		"OCR"
+				"tall"		"56"
+				"weight"	"600"
+				"range"		"0x0000 0x017F" //	Basic Latin, Latin-1 Supplement, Latin Extended-A
+				"yres"	"1536 1599"
```
Comment (AdamTadeusz) [inline mp/game/neo/resource/ClientScheme.res:1451]: mhm

File: mp/game/neo/resource/ClientScheme.res [hunk 6]
```diff
@@ -1614,7 +1719,27 @@ Scheme
...
+				"antialias"	"1"
+				//"additive"	"1"
+			}
+			"6"
+			{
+				"name"		"Green Mountain 3"
+				"tall"		"30"
+				"weight"	"900"
+				"range"		"0x0000 0x017F" //	Basic Latin, Latin-1 Supplement, Latin Extended-A
+				"yres"	"1536 1599"
```
Comment (AdamTadeusz) [inline mp/game/neo/resource/ClientScheme.res:1732]: yup still there

File: mp/game/neo/resource/ClientScheme.res [hunk 7]
```diff
@@ -1665,7 +1792,27 @@ Scheme
...
+				"antialias"	"1"
+				//"additive"	"1"
+			}
+			"6"
+			{
+				"name"		"Green Mountain 3"
+				"tall"		"41"
+				"weight"	"900"
+				"range"		"0x0000 0x017F" //	Basic Latin, Latin-1 Supplement, Latin Extended-A
+				"yres"	"1536 1599"
```
Comment (AdamTadeusz) [inline mp/game/neo/resource/ClientScheme.res:1805]: *burps*

File: mp/game/neo/resource/chatscheme.res [hunk 8]
```diff
@@ -349,6 +349,30 @@ Scheme
...
+				"weight"	"700"
+				"yres"	"1200 1440"
+				"dropshadow"	"1"
+			}
+			"6"
+			{
+				"name"		"Verdana"
+				"tall"		"20" //"24"
+				"weight"	"700"
+				"yres"	"1536 1600"
```
Comment (AdamTadeusz) [inline mp/game/neo/resource/chatscheme.res:365]: and finished

**Review by AdamTadeusz — CHANGES_REQUESTED** [review]

**Review by AdamTadeusz — DISMISSED** [review]

**Review by AdamTadeusz — APPROVED** [review]

**Review by nullsystem — APPROVED** [review]
Well, it didn't negatively affect 480p, 720p, and 1080p

**Pull request merged**